pkg/virt-controller/watch/node_test.go

```diff
@@ -147,34 +149,28 @@ var _ = Describe("Node controller with", func() {
 			node := NewHealthyNode("testnode")
 			node.Annotations[virtv1.VirtHandlerHeartbeat] = nowAsJSONWithOffset(-10 * time.Minute)
 
-			addNode(node)
-
 			kubeClient.Fake.PrependReactor("patch", "nodes", func(action testing.Action) (handled bool, obj runtime.Object, err error) {
 				patch, ok := action.(testing.PatchAction)
 				Expect(ok).To(BeTrue())
 				Expect(string(patch.GetPatch())).To(Equal(`{"metadata": { "labels": {"kubevirt.io/schedulable": "false"}}}`))
 				return true, nil, nil
 			})
 
-			vmiInterface.EXPECT().List(gomock.Any()).Return(&virtv1.VirtualMachineInstanceList{}, nil)
-
-			controller.Execute()
```

Comment:
I would prefer to test it with `controller.Execute()` if that's not too much of the work.

Reply:
updated to test with `controller.Execute() `